database.rules.json

```diff
@@ -73,6 +73,20 @@
             ".validate": "newData.isString() || !newData.exists()"
           }
         },
+        "ratings": {
+          "normal": {
+            ".write": "!data.exists() && newData.exists() && auth != null && newData.val() == 1200",
```

Comment:
I think this actually enforces a value of 1200 every time you write, so you can't ever update the rating at all.

Reply:
I don't remember whether I uploaded this exact version of the rules. I'll do so on Monday to make sure that it works, but as far as I understand it, these rules are only enforced on client-side updates, and not on server-side updates. The `!data.exists()` would otherwise also only allow a value to be written once.

Reply:
Ah I see, makes sense!